modules/thermal_hydraulics/doc/content/source/materials/ADWallHeatTransferCoefficientWolfMcCarthyMaterial.md

```diff
@@ -0,0 +1,15 @@
+# ADWallHeatTransferCoefficientWolfMcCarthyMaterial
+
+The material computes the convective heat transfer coefficient using the Wolf-McCarthy correlation  (see [!cite](wolf_mccarthy1960)).
+
+The Nusselt number is calculated as:
+
+\begin{equation}
+  Nu= 0.025 Re^{0.8} Pr^{0.4} \left(\frac{T_{fluid}}{T_{wall}}\right)^{-0.55} 
```

Comment:
I think you shouldn’t have the minus sign in the exponent of the bulk to wall. Any chance you could give an eq# or page number for the ref?

Reply:
I am using Equation 25 in this report. https://www.osti.gov/servlets/purl/4842792 . There are several forms in this report, but I think this is the one that is most commonly used.

Reply:
It’s a typo in the report? Wall over bulk is used everywhere else, even in figure 25 where the text says eq 25 is compared to experiments.

Reply:
You are right, this must be a typo in the report. I'll fix it. Thanks for the catch!

Reply:
Agreed, nice catch! I probably wouldn't have caught it in my review...

Lise, can you make a note in your doc where you reference that doc to mention that there is a typo in the reference?